pomegranate/utils.pyx

```diff
@@ -118,8 +118,9 @@ cdef double gamma(double x) nogil:
 	cdef double den, num, result, z, y
 	cdef int i, n, arg_was_less_than_one
 
-	if x == 0.0:
-		return INF
+	with gil:
```

Comment:
just realised it's comparing to zero, so `-1e-9 < x < 1e-9` might be an acceptable option.
thoughts?